en/docs/administer/logging-and-monitoring/monitoring/monitoring-server-health.md

```diff
@@ -239,17 +239,14 @@ Follow the instructions below to add a custom Health Checker:
 
     [carbon_health_check.health_checker.data_source_health_checker.properties]
     'monitored.datasources' = "jdbc/WSO2CarbonDB,jdbc/WSO2AM_DB,jdbc/SHARED_DB"
-    
-    [carbon_health_check.health_checker.data_source_health_checker]
-    enable =false
-    order = 98
```

Comment:
Shouldn't this be the configuration for super tenant health checker? Just like the configurations that come before this are for `carbon_health_check.health_checker.data_source_health_checker` and `carbon_health_check.health_checker.data_source_health_checker.properties`; I believe this should be a typo. We should change this from `[carbon_health_check.health_checker.data_source_health_checker]` to `[carbon_health_check.health_checker.super_tenant_health_checker]`

```
    [carbon_health_check.health_checker.super_tenant_health_checker]
    enable =false
    order = 98
```

Reply:
fixed with https://github.com/wso2/docs-apim/pull/8697/commits/f4b07aa1cacf3578d48860f553da4affdc2ace6e